lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless government_data_availible?
+
+      return GovukHealthcheck::WARNING if data_stale?
+
+      GovukHealthcheck::OK
+    end
+
+    def details
+      {
+        critical: "No government data availible",
+        warning: warning_details_content,
+      }
+    end
+
+    def enabled?
+      true
+    end
+
+  private
+
+    def government_data_availible?
+      BulkData::GovernmentRepository.new.cache_populated?
```

Comment:
It'd make sense to have the repository as an instance variable rather than creating a new instance each time. 

This would also make the code shorter so you could probably kill this method and instead have the guard clause be: `return GovukHealthcheck::CRITICAL unless repository.cache_populated?`

Reply:
Done, that really cleared it up, thanks